### PR TITLE
use semver; simple perl-versions workflow

### PR DESCRIPTION
**Pull request by happy-barney**
Provide simple reusable workflow for basic usage of `perl-versions` action.

Use `semver` package to safely compare versions.

### CODE REVIEW

File: .github/workflows/perl-versions.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  perl-versions:
+    runs-on: ubuntu-latest
+    name: List perl versions
+    outputs:
+      perl-versions: ${{ steps.action.outputs.perl-versions }}
+    steps:
+      - name: Perl versions action step
+        id: action
+        uses: perl-actions/perl-versions@main
```
Comment (atoomic) [inline .github/workflows/perl-versions.yml:24]: is using `@main` the right thing to do there?

File: .github/workflows/perl-versions.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: List perl versions
```
Comment (atoomic) [inline .github/workflows/perl-versions.yml:1]: should we consider running that workflow on PR for sanity check?

**Review by atoomic — APPROVED** [review]

File: .github/workflows/perl-versions.yml (hunk 1 above)
Comment (happy-barney) [reply .github/workflows/perl-versions.yml:24]: yes, we don't have tags yet

File: .github/workflows/perl-versions.yml (hunk 2 above)
Comment (happy-barney) [reply .github/workflows/perl-versions.yml:1]: I'm thinking about that so far no idea (but I'm sure I will come with some)
so far I have (private) repo where I call workflows with manual dispatch.

**Pull request merged**